admin/meeting/20170321.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: meeting
+sitemap: false
+description: Meeting
+date: 2017-03-21
+time:
+  open: "20:00"
+  close: "20:15"
+members:
+ - Dave Seff
+ - Phil Gowenlock
+ - James Churchill
+ - Alex Wixted
+ - Josh Hogendorn
+ - David Bussenschott
+ - Brendon Halliday
+ - Meg Drinkswater
+ - Meka Beecham
+ - Mike Ando
+ - Sven Hanzka
+ - Steph Piper
+ - Karl Richardson
+ - Rosa Tong
+ - Jeremy Robinson
+ - Lysa Singh
+ - Michael King
+ - Tim Reichie
+ - Doug Miller
+ - Luke Ramage
+ - Luke Davies
+ - Eris Ryan
+ - Angie Yoo
+Scott Wilson
+Jaimyn Mayer
+
+author: Dave Seff
+signed: PRESIDENT OR CHAIRPERSON OF THE MEETING
+---
+
+## Agenda
+
+- Purchase Proposal: Step Ladder $842
+- Safety Policy
+- Electrical Safety
+- Vote to reimburse Dave for Metal Shop Storage Drawers ($450)
+- World Science Festival
+- Starlight Children's Foundation Workshop; Saturday 8th April, 12:30 - 3:30pm
+
+## Meeting Notes
+
+Volunteers for WSF please see Steph. Need ideas for demo material ( 3d print and other projects)
+
+Safety Policy in effect:
```

Comment:
This copy and pasted safety wording is not the one that was voted through, and is misleading to put it in the notes.